src/applications/edu-benefits/10203/submitForm.js

```diff
@@ -1,17 +1,36 @@
 import { submitToUrl } from 'platform/forms-system/src/js/actions';
 import { transformForSubmit } from 'platform/forms-system/src/js/helpers';
+import _ from 'lodash';
 
 const submitForm = (form, formConfig) => {
   const body = formConfig.transformForSubmit
     ? formConfig.transformForSubmit(formConfig, form)
     : transformForSubmit(formConfig, form);
+  const selectedBenefits = _.get(form.data, 'view:benefit', {});
+  let concatenatedBenefits = '';
+  Object.keys(selectedBenefits).forEach(function(key) {
+    if (selectedBenefits[key]) {
+      if (concatenatedBenefits === '') {
+        concatenatedBenefits = key;
+      } else {
+        concatenatedBenefits += `, ${key}`;
+      }
+    }
+  });
```

Comment:
Not crazy about this. Has to be a better way.

Reply:
In AC 1 they are asking for a concatenated message divided by commas. The values are stored as an array of json objects where the values they want concatenated are the keys.  A foreach seemed like the best way to do it.